misc/de-metas-common/de-metas-common-product/src/main/java/de/metas/common/product/v2/request/constants/SwaggerDocConstants.java

```diff
@@ -24,9 +24,10 @@
 
 public class SwaggerDocConstants
 {
-	public static final String PRODUCT_IDENTIFIER_DOC = "Identifier of the product in question. Can be\n"
+	public static final String PRODUCT_IDENTIFIER_DOC = "Identifier of the product for the price in question. Can be\\n"
```

Comment:
i superficially looked at the places where this constant is used, and i think that "for the price" doesn't really fit..or did i check too briefly?

Reply:
yes, you're right. my mistake